### PR TITLE
Updated HTML

### PR DESCRIPTION
**Pull request by nicoletr**
Updated some minor parts of the HTML

### CODE REVIEW

**Review by MA-FS — CHANGES_REQUESTED** [review]
Can we please add this to the body
    <select id="list-dropdown" name="list">
    </select>

File: index.html [hunk 1]
```diff
@@ -12,6 +12,13 @@
 <body>
     <main>    
         <h1>Finance Tracker</h1>
+        <!--Form goes here-->
+
```
Comment (MA-FS) [inline index.html:16]: Can we please add this to the body
    <select id="list-dropdown" name="list">
    </select>

Comment (MA-FS) [inline index.html:16]:
```
    <select id="list-dropdown" name="list">
    </select>
```

**Review by PamelaW80 — APPROVED** [review]

**Review by MA-FS — CHANGES_REQUESTED** [review]
Please check code

**Review by moeac — APPROVED** [review]

**Review by PamelaW80 — APPROVED** [review]

**Review by MA-FS — APPROVED** [review]
Looks good

**Review by PamelaW80 — APPROVED** [review]

**Pull request merged**